multisig/src/setup.rs

```diff
@@ -47,38 +47,20 @@ pub trait SetupModule:
     #[endpoint(deployChildContracts)]
     fn deploy_child_contracts(
         &self,
-        egld_esdt_swap_code: BoxedBytes,
         multi_transfer_esdt_code: BoxedBytes,
         esdt_safe_code: BoxedBytes,
         price_aggregator_contract_address: Address,
         esdt_safe_eth_tx_gas_limit: Self::BigUint,
         multi_transfer_esdt_eth_tx_gas_limit: Self::BigUint,
-        wrapped_egld_token_id: TokenIdentifier,
-        #[var_args] token_whitelist: VarArgs<TokenIdentifier>,
     ) -> SCResult<()> {
         // since contracts can either be all deployed or none,
         // it's sufficient to check only for one of them
         require!(
-            self.egld_esdt_swap_address().is_empty(),
+            self.esdt_safe_address().is_empty(),
```

Comment:
why don't we take this function out completely ? as we could deploy the child contract with a script and give ownership to this contract. plus we must use deployfromSource or upgradeFromSource - for this call 600million gas is not enough.

Reply:
Okay. Will do it in the next PR.